src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -16,6 +19,26 @@ public SelectTestFigureOptionListener(DriverManager driverManager) {
 
 	@Override
 	public void actionPerformed(ActionEvent e) {
-		FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+		DriverCommand command;
+		switch (e.getActionCommand()) {
+			case "Figure Joe 1":
+				FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+				break;
+			case "Figure Joe 2":
+				FiguresJoe.figureScript2(driverManager.getCurrentDriver());
+				break;
+			case "Figure Square":
+				command = FigureFactory.getSquare();
+				command.execute(driverManager.getCurrentDriver());
+				break;
+			case "Figure Rectangle":
+				command = FigureFactory.getRectangle();
+				command.execute(driverManager.getCurrentDriver());
+				break;
+			case "Figure Triangle":
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.